specification/iotoperations/IoTOperations.Management/models/instance.tsp

```diff
@@ -117,12 +114,6 @@ model DataFlowInstanceProperties {
   enabled: boolean;
```

Comment:
we should decorate all these enabled fields with `@visibility("create", "read")`

https://typespec.io/docs/standard-library/built-in-decorators#@visibility

Reply:
also, should we consider having these also use the `OperationalMode` enum instead of being boolean type?

Reply:
I'll bring this up with Ro, but I agree. we should align with the rest of the RP